### PR TITLE
Basic setup for frontend

### PR DESCRIPTION
**Pull request by ninnivictoria**
- Added vuetify and table for the leaderboard
- Created basic routing with landing page, leaderboard and resources
- Put mock data in leaderboard table
- Made leaderboard responsive-ish

### CODE REVIEW

**Comment by flxwu** [comment]
puh this is huge

File: frontend/src/router.js [hunk 1]
```diff
@@ -12,13 +12,22 @@ export default new Router({
...
       component: () =>
-        import(/* webpackChunkName: "about" */ "./views/About.vue")
+        import(/* webpackChunkName: "about" */ "./views/Leaderboard.vue")
+    },
+    {
+      path: "/resources",
+      name: "resources",
+      // route level code-splitting
+      // this generates a separate chunk (about.[hash].js) for this route
+      // which is lazy-loaded when the route is visited.
```
Comment (flxwu) [inline frontend/src/router.js]: we don't need this comment twice do we 😂

File: frontend/src/router.js [hunk 2]
```diff
@@ -12,13 +12,22 @@ export default new Router({
...
-      path: "/about",
-      name: "about",
+      path: "/leaderboard",
+      name: "leaderboard",
       // route level code-splitting
       // this generates a separate chunk (about.[hash].js) for this route
       // which is lazy-loaded when the route is visited.
       component: () =>
-        import(/* webpackChunkName: "about" */ "./views/About.vue")
+        import(/* webpackChunkName: "about" */ "./views/Leaderboard.vue")
```
Comment (flxwu) [inline frontend/src/router.js]: the webpackChunkName doesn't really make sense

File: frontend/src/router.js [hunk 3]
```diff
@@ -12,13 +12,22 @@ export default new Router({
...
+    },
+    {
+      path: "/resources",
+      name: "resources",
+      // route level code-splitting
+      // this generates a separate chunk (about.[hash].js) for this route
+      // which is lazy-loaded when the route is visited.
+      component: () =>
+        import(/* webpackChunkName: "about" */ "./views/Resources.vue")
```
Comment (flxwu) [inline frontend/src/router.js]: this webpackChunkName makes even less sense as it disables code-splitting

File: frontend/src/views/Home.vue [hunk 4]
```diff
@@ -1,18 +1,29 @@
 <template>
-  <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+  <div class="page">
+    <!-- TODO: Rewrite text, make button better. Update with find out more link. -->
+    <img 
+      class="hacktoberfest" 
+      src="/hacktoberfest.svg" 
+      alt="">
```
Comment (flxwu) [inline frontend/src/views/Home.vue]: this is an excellent alternative text

File: frontend/src/views/Home.vue [hunk 5]
```diff
@@ -1,18 +1,29 @@
...
-// @ is an alias to /src
-import HelloWorld from "@/components/HelloWorld.vue";
-
-export default {
-  name: "home",
-  components: {
-    HelloWorld
-  }
-};
+export default {};
```
Comment (flxwu) [inline frontend/src/views/Home.vue]: whut

Comment (flxwu) [reply frontend/src/views/Home.vue]: is there a reason for this? 🤔

File: frontend/src/views/Home.vue [hunk 6]
```diff
@@ -1,18 +1,29 @@
...
 </script>
+
+<style lang="stylus">
+
```
Comment (flxwu) [inline frontend/src/views/Home.vue]: UL

File: frontend/src/views/Resources.vue [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<template>
+  <div class="page">
+    <div class="content white--text">
+      <h1 class="white--text">Resources</h1>
+      <p>Find out how you can get the most out of Hacktoberfest 2018</p>
+    </div>
+  </div>
+  <!-- TODO: Add projects to contribute to based on skill level, link to websites (like maintainerswanted.com and opensource.guide) -->
+</template>
```
Comment (flxwu) [inline frontend/src/views/Resources.vue:9]: CRLF

File: frontend/public/index.html [hunk 8]
```diff
@@ -5,7 +5,9 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title>frontend</title>
+    <title>Hacktoberfest Frankfurt</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
```
Comment (flxwu) [inline frontend/public/index.html:9]: I think I'm missing it, but where are you using Roboto? 🙈

File: frontend/package.json [hunk 9]
```diff
@@ -9,17 +9,25 @@
   },
   "dependencies": {
     "socket.io-client": "^2.1.1",
+    "@babel/polyfill": "^7.0.0-rc.1",
     "vue": "^2.5.17",
+    "vue-mq": "^0.2.1",
     "vue-router": "^3.0.1",
+    "vuetify": "^1.2.0",
     "vuex": "^3.0.1"
```
Comment (flxwu) [inline frontend/package.json:17]: are you going to use vuex?

File: frontend/package.json [hunk 10]
```diff
@@ -9,17 +9,25 @@
...
   },
   "devDependencies": {
+    "@fortawesome/fontawesome-free": "^5.3.1",
     "@vue/cli-plugin-babel": "^3.0.4",
     "@vue/cli-plugin-eslint": "^3.0.4",
     "@vue/cli-service": "^3.0.4",
     "@vue/eslint-config-prettier": "^3.0.4",
+    "babel-plugin-transform-imports": "^1.5.0",
     "node-sass": "^4.9.0",
     "sass-loader": "^7.0.1",
```
Comment (flxwu) [inline frontend/package.json:27]: are you writing sass? I don't see any sass files 👀

File: frontend/public/index.html (hunk 8 above)
Comment (flxwu) [reply frontend/public/index.html:9]: And also, do we need all these thicknesses?

**Review by flxwu — DISMISSED** [review]

File: frontend/src/views/Home.vue (hunk 5 above)
Comment (ninnivictoria) [reply frontend/src/views/Home.vue]: Nope 😂

File: frontend/src/views/Home.vue (hunk 4 above)
Comment (ninnivictoria) [reply frontend/src/views/Home.vue]: 😂

File: frontend/public/index.html (hunk 8 above)
Comment (ninnivictoria) [reply frontend/public/index.html:9]: Vuetify uses roboto and the whole website is using it now

File: frontend/package.json (hunk 9 above)
Comment (ninnivictoria) [reply frontend/package.json:17]: If we need it, yes. We'll see

File: frontend/package.json (hunk 10 above)
Comment (ninnivictoria) [reply frontend/package.json:27]: I'm going to write sass from now on

File: frontend/public/index.html (hunk 8 above)
Comment (flxwu) [reply frontend/public/index.html:9]: thanks for the clarification! 😊

File: frontend/package.json (hunk 10 above)
Comment (ninnivictoria) [reply frontend/package.json:27]: Stylus 😤

File: frontend/package.json (hunk 9 above)
Comment (flxwu) [reply frontend/package.json:17]: 👍

File: frontend/src/App.vue [hunk 11]
```diff
@@ -1,29 +1,45 @@
...
-#nav {
-  padding: 30px;
-  a {
-    font-weight: bold;
-    color: #2c3e50;
-    &.router-link-exact-active {
-      color: #42b983;
-    }
+<script>
+import TheNavigation from './components/TheNavigation.vue'
```
Comment (flxwu) [inline frontend/src/App.vue]: are you now writing with or without semicolons? this line doesn't have them but the next line does ^^

**Review by flxwu — APPROVED** [review]
LGTM, we should align the object scheme before merging, 

```javascript
{
    "name": "flxwu",
    "prs": 4,
    "latestPr": "2018-10-07",
    "latestProject": "cssfrankfurt/hacktoberfest2018"
  },
```
is what https://hacktoberfest-frankfurt.herokuapp.com/api/data currently returns.

**Pull request merged**